orchestra/tests/helpers/fixtures.py

```diff
@@ -235,7 +240,8 @@ def setup_models(test_case):
     for task_slug, details in tasks.items():
         task = TaskFactory(project=test_case.projects[details['project_name']],
                            step_slug=test_case.test_step_slug,
-                           status=details['status'])
+                           status=details['status'],
+                           start_datetime=parse('2015-10-12T01:00:00+00:00'))
```

Comment:
https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.timezone.make_aware would be my preference (no argument necessary for `make_aware`) :-)

But feel free to overrule if you don't like my django-utils-fanboyism.